lib/phoenix/naming.ex

```diff
@@ -53,6 +53,25 @@ defmodule Phoenix.Naming do
   end
 
   @doc """
+  Inflects the default name of the PubSub server from the module name
+
+  ## Examples
+
+      iex> Phoenix.Naming.module_to_pub_server(MyApp.MyChannel)
+      MyApp.PubSub
+
+      iex> Phoenix.Naming.module_to_pub_server(MyApp.Admin.MyChannel)
+      MyApp.PubSub
+
+  """
+  def module_to_pub_server(mod) do
```

Comment:
It would be nice to make this generic passing the PubSub as argument because there are other places we could use it, like here: https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/controller.ex#L729-L732 :)